rubrik_cdm/cluster.py

```diff
@@ -40,24 +39,21 @@ def cluster_version(self, timeout=15):
             'cluster_version: Getting the software version of the Rubrik cluster.')
         return self.get('v1', '/cluster/me/version', timeout=timeout)['version']
 
-    def cluster_version_check(self, minimum_cluster_version, timeout=15):
-        """Determine if the Rubrik cluster is using running an earlier release than the provided CDM `minimum_cluster_version`.
-        If the CDM version is an earlier release than the "clusterVersion", the following message error message is thrown:
-            Error: The Rubrik cluster must be running CDM version {`minimum_cluster_version`} or later.
+    def minimum_installed_cdm_version(self, cluster_version, timeout=15):
+        """Determine if the Rubrik cluster is running the provided CDM `cluster_version` or later. If the cluster is running an earlier release
+        of CDM, `False` is returned. If the cluster is running the provided `cluster_version`, or a later release, `True` is returned.
 
         Arguments:
-            minimum_cluster_version {float} -- The minimum required version of Rubrik CDM you wish ensure is running.
+            cluster_version {float} -- The minimum required version of Rubrik CDM you wish ensure is running.
 
         Keyword Arguments:
             timeout {int} -- The number of seconds to wait to establish a connection the Rubrik cluster before returning a timeout error. (default: {15})
-
-        Raises:
-            CDMVersionException -- When the Rubrik cluster is using running an earlier release than the provided CDM `minimum_cluster_version` the following error is raised:
-            The Rubrik cluster must be running CDM version {`minimum_cluster_version`} or later.
         """
 
-        if float(self.cluster_version(timeout)[:3]) < float(minimum_cluster_version):
-            raise CDMVersionException(float(minimum_cluster_version))
+        if float(self.cluster_version(timeout)[:3]) < float(cluster_version):
```

Comment:
Shouldn't this be <= ?

Reply:
And since this is a boolean operation, you can shorten the expression like this: 

return float(self.cluster_version(timeout)[:3]) <= float(cluster_version)

Reply:
`Shouldn't this be <= ?` 

`<` is all we need. The functions purpose is to "make sure the cluster is at least running this version of CDM" so we only need to check to see if the version is lower than the provided `cluster_version`.

For example, when connecting to a cluster running CDM v5.0 `rubrik.minimum_installed_cdm_version(5.0)` should return `True`. If you switch the logic to `<=` it will return `False`.
___

`return float(self.cluster_version(timeout)[:3]) <= float(cluster_version)`

We want to make sure that someone just learning Python can very easily understand the code so it's not uncommon for us to be more verbose than needed. The current implementation fits this criteria better than just the straight `return` statement.